filebeat/tests/system/config/filebeat.yml.j2

```diff
@@ -105,6 +105,10 @@ filebeat.config.{{ reload_type|default("inputs") }}:
   {% endif -%}
 {% endif -%}
 
+{% if logging_level -%}
```

Comment:
If we add a config option for logging level we should do it here: https://github.com/elastic/beats/blob/master/libbeat/tests/system/config/libbeat.yml.j2

But I don't think this is needed as python catches the output from `-d "*"` and does not use the logs.